modules/statics/src/coins.ts

```diff
@@ -9594,6 +9610,24 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset.FTT,
     AccountCoin.DEFAULT_FEATURES
   ),
+  solToken(
+    'f194cd14-d0c2-443a-b717-e826143c7e33',
+    'sol:srm-wormhole',
+    'Serum (Wormhole)',
+    6,
+    'xnorPhAzWXUczCP3KjU5yDxmKKZi5cSbxytQ1LgE3kG',
+    UnderlyingAsset.SRM,
+    AccountCoin.DEFAULT_FEATURES
+  ),
+  solToken(
+    'cd7d9622-1467-405d-af1f-09edd1d431d6',
+    'sol:usdc-wormhole',
+    'USD Coin (PoS) (Wormhole)',
```

Comment:
```suggestion
    'USD Coin (PoS) (Wormhole) (USDC)',
```
Do we need to match solscan's exact naming?

Reply:
Seems like solscan appends the token symbol while displaying the name, we generally do not include that in the name so I left it out as well